test/unit/Reflector/GenericTest.php

```diff
@@ -160,6 +160,8 @@ public function testReflectWithAggregateSourceLocatorWhenIdentifierDoesNotExist(
 
     public function testReflectWithAggregateSourceLocatorFindsClass()
     {
+        $this->markTestIncomplete('See https://github.com/Roave/BetterReflection/issues/43');
```

Comment:
The point is that string source locators don't "locate", but "provide". If they were to "locate" (and therefore return `null` or a `LocatedSource`) then this would be fine.